### PR TITLE
update ODS releases on start page

### LINKED ISSUE #115: Roadmap wrong on home page
On page http://www.opendevstack.org/index.html:

Roadmap
June 2019 - release 1.1
October 2019 - release 2.0
Following: Release of continuously advanced versions

The 2.0 version is planned for December I think and 1.2 is missing.

### PR DESCRIPTION
**Pull request by tbugfinder**
fix #115

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,7 +36,9 @@ <h5 class="card-title">Main advantages</h5>
         <h5 class="card-title">Roadmap</h5>
         <ul>
           <li>June 2019 - release <a href="https://github.com/orgs/opendevstack/projects/4">1.1</a></li>
-          <li>October 2019 - release <a href="https://github.com/orgs/opendevstack/projects/6">2.0</a></li>
+          <li>October 2019 - release <a href="https://github.com/orgs/opendevstack/projects/8">1.2</a></li>
+          <li>December 2019 - release <a href="https://github.com/orgs/opendevstack/projects/6">2.0</a></li>
+          <li>March 2020 - release <a href="https://github.com/orgs/opendevstack/projects/9">3.0</a></li>
```
Comment (metmajer) [inline index.html]: @tbugfinder as far as I know, 3.0 will come in June :)

Comment (tbugfinder) [reply index.html]: ok, the GitHub project has set: `Release date: May 2020`.
Is there a version / release tag expected for end-of-march?

Comment (tbugfinder) [reply index.html]: Hi @metmajer,
would you support using Github project release date: May 2020? How about a tagged intermediate release end-of-march?

Comment (clemensutschig) [reply index.html]: we do not expect a march release ...

Comment (tbugfinder) [reply index.html]: updated it to May matching the Github project definition.

**Review by michaelsauter — APPROVED** [review]

**Comment by tbugfinder** [comment]
ping

**Review by stefanlack — APPROVED** [review]
LGTM

**Pull request merged**